src/omnicore/omnicore.cpp

```diff
@@ -915,7 +920,9 @@ int parseTransaction(bool bRPConly, const CTransaction &wtx, int nBlock, unsigne
   uint64_t outAll = 0;
   uint64_t txFee = 0;
   int omniClass = 0;
-  mp_tx->Set(wtx.GetHash(), nBlock, idx, nTime);
+  if (NULL != mp_tx) {
+      mp_tx->Set(wtx.GetHash(), nBlock, idx, nTime);
+  }
 
```

Comment:
These shouldn't ever be NULL but I see the idea behind safety, checking != NULL, but perhaps it's worth logging if they do fail that check?